index.html

```diff
@@ -2403,7 +2405,7 @@ <h2 class="header">Year</h2>
 								<p>Pea horseradish azuki bean lettuce avocado asparagus okra. Kohlrabi radish okra azuki bean corn fava bean mustard tigernut jÃ­cama green bean celtuce collard greens avocado quandong fennel gumbo black-eyed pea. Grape silver beet watercress potato tigernut corn groundnut. Chickweed okra pea winter purslane coriander yarrow sweet pepper radish garlic brussels sprout groundnut summer purslane earthnut pea tomato spring onion azuki bean gourd. Gumbo kakadu plum komatsuna black-eyed pea green bean zucchini gourd winter purslane silver beet rock melon radish asparagus spinach. </p>
 								<p>Beetroot water spinach okra water chestnut ricebean pea catsear courgette summer purslane. Water spinach arugula pea tatsoi aubergine spring onion bush tomato kale radicchio turnip chicory salsify pea sprouts fava bean. Dandelion zucchini burdock yarrow chickpea dandelion sorrel courgette turnip greens. </p>
 							</div>
-							<div class="step-pane sample-pane alert-danger alert" data-step="3">
+							<div class="step-pane sample-pane alert-danger alert" data-step="3" style="position: relative;">

```

Comment:
Ah, this was me testing that overflow was no longer clipped via some quick inline hackery and forgetting to remove it once I was done.